pyproject.toml

```
@@ -39,6 +39,7 @@ dependencies = [
     "ipypopout>=0.0.11",
     "astroquery",
     "requests",
+    "importlib_metadata==7.2.1"
```

Is this more future-proof?
```suggestion
    "importlib_metadata!=8.0.0"
```